c2corg_common/attributes.py

```diff
@@ -644,3 +645,85 @@
     'tourism',
     'magazine'
 ]
+
+event_types = [
+    'avalanche',
+    'stone_fall',
+    'falling_ice',
+    'other_fall',
```

Comment:
I know ``other_fall`` is the value from v5:
https://github.com/c2corg/camptocamp.org/blob/72e777075f2e6260b1d2ef563ab0f90c383038ba/apps/frontend/modules/xreports/config/module.yml#L17
but it's too confusing and should be renamed here and in the migration script https://github.com/c2corg/v6_api/blob/reports/c2corg_api/scripts/migration/documents/reports.py#L154

The french translation is "chute d'une personne ou de la cordée" so I suggest to use ``person_fall`` instead.